index.css

```diff
@@ -847,34 +848,119 @@ button :disabled {
   background: #f4f1f0;
 }
 
-.other_works_item :hover + .action {
+.required::before {
+  content: "*";
+  color: red;
+  float: right;
+}
+
+/* The Modal (background) */
+#myModal {
+  display: none;
+  position: fixed;
+  z-index: 1;
+  padding-top: 100px;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: rgb(0, 0, 0);
+  background-color: rgba(0, 0, 0, 0.4);
+}
+
+/* Modal Content */
+.modal_content {
+  background-color: #fefefe;
+  margin: auto;
+  padding: 20px;
+  border: 1px solid #888;
+  width: 80%;
+}
```

Comment:
[DESKTOP] Your popup window is working perfectly, well done! Please make sure to also follow the original design:
| Figma design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/209198232-06b63cad-0c37-428b-bdae-40ba1f26a3a5.png)  | ![image](https://user-images.githubusercontent.com/42799579/209198181-a7f85e77-6749-4c7c-a7cc-e39dcd1c4c32.png)  |

Please notice that I highlighted the margin between your buttons and the bottom of the popup 😉.
- [ ] Kindly make sure to adjust the button to be displayed at the bottom of your popup, instead of below the text.